src/aframe-streetmix-parsers.js

```diff
@@ -813,35 +808,56 @@ function processBuildings (left, right, streetWidth, showGround, length) {
 
     if (showGround) {
       var groundJSONString = JSON.stringify(streetmixParsersTested.createGroundArray(currentValue));
+      if ("[]" !== groundJSONString){
+        let mixin = streetmixParsersTested.createGroundArray(currentValue)[0].mixin;
+        repeat = length / 30;
+        if (mixin == "ground-parking-lot"){
+          groundJSONString = '[{"tag":"a-entity","position":"0 -0.2 0","mixin":"' + mixin +  '","geometry":"primitive:plane;height:' + length + ';width:40","material":"repeat:.5 ' + repeat + ';roughness:1"}]';
```

Comment:
It seems like this is rewriting values provided in the definitions of ground mixins in assets.js. These mixins aren't used elsewhere, it's for this purpose. So it might make sense to make as many changes as you can to assets.js and then limit changes here to dynamic values only when needed

Reply:
I added a few commits that should address this by updating `buildingsArray.forEach` in `aframe-streetmix-parsers.js` and `assets,js`